helm/kanister-operator/values.yaml

```diff
@@ -53,3 +53,5 @@ resources:
 # requests:
 #  cpu: 100m
 #  memory: 128Mi
+metrics:
```

Comment:
Should we add this as a sub-value under `controller` above?
Additionally, add a comment

Reply:
Sure. Makes sense. I've made it as a sub-value under `controller`. I've added a comment.
One more point - I'm maintaining two levels of hierarchy for the helm flags - like `metrics.enabled` instead of `metrics_enabled` because we may want to add more flags under `metrics` hierarchy in the future if we consider adding labels for a new metric partially.

Reply:
Yes, that's the right way to do it 👍🏼

Reply:
do we just want to set the env var in the pod as part of this PR? Do we not want to use it anywhere?